File: app/src/main/java/com/hamza/Wazaker/Data_sets/AzkarListDataset.java
package com.hamza.Wazaker.Data_sets;

import com.hamza.Wazaker.Models.AzkarListModel;

import java.util.ArrayList;
import java.util.List;

public class AzkarListDataset {

    public static List<AzkarListModel> azkarList;

    public static List<AzkarListModel> getAzkarList() {

        azkarList = new ArrayList<>();

        azkarList.add(new AzkarListModel("أذكار المسجد"));
        azkarList.add(new AzkarListModel("أذكار بعد الصلاة "));
        azkarList.add(new AzkarListModel("أذكار الأذان "));
        azkarList.add(new AzkarListModel("أذكار الوضوء"));
        azkarList.add(new AzkarListModel("الأدعيـة القرآنية"));
        azkarList.add(new AzkarListModel("الأدعية النبوية "));
        azkarList.add(new AzkarListModel("أذكار الطعام"));
        azkarList.add(new AzkarListModel("أذكار المنزل"));
        azkarList.add(new AzkarListModel("أذكار النوم"));
        azkarList.add(new AzkarListModel("أذكار الإستيقاظ من النوم"));
        azkarList.add(new AzkarListModel("أذكار عظيمة"));
        azkarList.add(new AzkarListModel("دعاء السفر"));
        azkarList.add(new AzkarListModel("دعاء ختم القرآن الكريم"));
        azkarList.add(new AzkarListModel("فضل الدعاء"));
        azkarList.add(new AzkarListModel("فضل تلاوة القرآن الكريم"));
        azkarList.add(new AzkarListModel("فضل الأذكار"));
        azkarList.add(new AzkarListModel("الرقية الشرعية"));
        azkarList.add(new AzkarListModel("أدعية للمتوفّي"));

        return azkarList;
    }}

File: app/src/main/java/com/islamey/hamza/wazaker/domain/DataSets/AzkarListDataset.kt
package com.islamey.hamza.wazaker.domain.DataSets

import android.content.Context
import com.islamey.hamza.wazaker.domain.Models.AzkarListModel
import com.islamey.wazkar.R

object AzkarListDataset {
    private lateinit var azkarList: MutableList<AzkarListModel>

    fun getAzkarList(context: Context): List<AzkarListModel> {
        azkarList = ArrayList()
        azkarList.add(AzkarListModel(context.getString(R.string.majed_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.after_salah_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.azan_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.wodoa_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.qurany_doaa)))
        azkarList.add(AzkarListModel(context.getString(R.string.naboya_doaa)))
        azkarList.add(AzkarListModel(context.getString(R.string.food_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.house_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.sleep_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.wakeup_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.great_azkar)))
        azkarList.add(AzkarListModel(context.getString(R.string.traveling_doaa)))
        azkarList.add(AzkarListModel(context.getString(R.string.quran_completed_doaa)))
        azkarList.add(AzkarListModel(context.getString(R.string.doaa_fdel)))
        azkarList.add(AzkarListModel(context.getString(R.string.quran_reading_fadl)))
        azkarList.add(AzkarListModel(context.getString(R.string.azkar_fadl)))
        azkarList.add(AzkarListModel(context.getString(R.string.roqia)))
        azkarList.add(AzkarListModel(context.getString(R.string.dead_doaa)))

        return azkarList
    }
}